pkg/identityserver/store/gateway.go

```diff
@@ -65,14 +65,15 @@ type Gateway struct {
 
 	Antennas []GatewayAntenna
 
-	LBSLNSSecret []byte `gorm:"type:BYTEA;column:lbs_lns_secret"`
+	LBSLNSSecret            []byte `gorm:"type:BYTEA;column:lbs_lns_secret"`
+	ClaimAuthenticationCode []byte `gorm:"type:BYTEA;column:claim_authentication_code"`
```

Comment:
I think I'd prefer to store the valid_from and valid_to in separate columns. I can imagine that in the future we'll want to be able to query "claimable" gateways, which is impossible when using a blob for the entire message.